integration_test/scripts/run-store-scp-and-wait.sh

```diff
@@ -1,9 +1,12 @@
 #!/bin/bash
 # run_step scp_port scp_finish_port
-cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-storescp
+export PATH=/opt/apache-maven-3.6.3/bin:$PATH
+cd /workspace/dcm4che/dcm4che-tool/dcm4che-tool-storescp &&
 mvn exec:java -ntp -Dexec.mainClass=org.dcm4che3.tool.storescp.StoreSCP -Dexec.args='--bind STORESCP@'$1':'$2' --directory /workspace/integration_test/storescp-data --accept-unknown' &
 apt-get -qq update
 apt-get -qq install -y netcat-openbsd
 echo 'Waiting for connection on port '$3' to finish store-scp step'
-nc -l -p $3
+nc -l -p $3 &
```

Comment:
&& is needed so that if the commands are executed one by one, and if the result of one of the commands is an error, the entire script will be completed with an error. Without &&, the bash script will continue with the next command regardless of the result of the previous one.